test/fileSummaryV1/integration_fileSummaryV1.test.js

```diff
@@ -0,0 +1,162 @@
+        keyFile: 'automation/firecloud-account.pem',
+        email: 'firecloud-dev@broad-dsde-dev.iam.gserviceaccount.com',
+        sub: 'ron.weasley@test.firecloud.org',
+        scope: 'profile email openid https://www.googleapis.com/auth/devstorage.full_control https://www.googleapis.com/auth/cloud-platform'
+    }).getToken();
+    authorizedToken = await new GoogleToken({
+        keyFile: 'automation/firecloud-account.pem',
+        email: 'firecloud-dev@broad-dsde-dev.iam.gserviceaccount.com',
+        sub: 'hermione.owner@test.firecloud.org',
+        scope: 'profile email openid https://www.googleapis.com/auth/devstorage.full_control https://www.googleapis.com/auth/cloud-platform'
+    }).getToken();
+
+    await postJsonTo(fenceAuthLink, "Bearer " + authorizedToken);
+});
+
+test.cb('integration_fileSummaryV1 responds with 400 if a uri is not provided', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .send({ notValid: 'dos://broad-dsp-dos.storage.googleapis.com/dos.json' })
+        .expect((response) => {
+            assert.strictEqual(response.statusCode, 400, "Incorrect status code")
+            assert(response.text.includes('must specify the URI'), "Received the wrong error message")
+        })
+        .end((error, response) => {
+            if (error) { t.log(response.body) };
+            t.end(error);
+        });
+});
+
+test.cb('integration_fileSummaryV1 responds with 400 if uri passed is malformed', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .send({ uri: 'somethingNotValidURL' })
+        .expect((response) => {
+            assert.strictEqual(response.statusCode, 400, "Incorrect status code")
+            assert(response.text.includes('must specify the URI'), "Received the wrong error message")
+        })
+        .end((error, response) => {
+            if (error) { t.log(response.body) };
+            t.end(error);
+        });
+});
+
+test.cb('integration_fileSummaryV1 responds with 401 if uri is valid but no authorization is provided', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .send({ uri: 'dos://dg.4503/preview_dos.json' })
+        .expect((response) => {
+            assert.strictEqual(response.statusCode, 401, "Incorrect status code")
+            assert(response.text.includes('must contain a bearer token'), "Received the wrong error message")
+        })
+        .end((error, response) => {
+            if (error) { t.log(response.body) };
+            t.end(error);
+        });
+});
+
+test.cb('integration_fileSummaryV1 responds with 200 and file metadata but no signed url if dos uri is valid, but an invalid bearer token is provided', (t) => {
+    supertest
+        .post('/fileSummaryV1')
+        .set('Content-Type', 'application/json')
+        .set('Authorization', `Bearer badToken`)
```

Comment:
I think this one and the next are sort of duplicates.  For this test, I suggest you just remove this line and test an otherwise "good" request to `fileSummaryV1` but without a bearer token.